eva/executor/function_scan_executor.py

```diff
@@ -37,14 +39,41 @@ def __init__(self, node: FunctionScanPlan):
     def validate(self):
         pass
 
+    def persistCost(self, name, type, frame_count, resolution, time):
+        # TODO: compute cost using time
+        cost = time
+        catalog_manager = CatalogManager()
+        catalog_manager.insert_udf_cost_catalog_entry(
+            name, type, cost, frame_count, resolution
+        )
+
     def exec(self, *args, **kwargs) -> Iterator[Batch]:
         assert (
             "lateral_input" in kwargs
         ), "Key lateral_input not passed to the FunctionScan"
         lateral_input = kwargs.get("lateral_input")
+        catalog_manager = CatalogManager()
         try:
             if not lateral_input.empty():
+                t_start = time()
```

Comment:
Use the inbuilt timer class. https://github.com/georgia-tech-db/eva/blob/f362dd4be87ddaa91b8ca0567c3c39a9fcd31302/eva/utils/timer.py

Reply:
Done